docs/contribute.md

```diff
@@ -1,6 +1,18 @@
-## Contribution Guide
+---                                                                             
+layout: default                                                                 
+title: Development Guide
+nav_order: 3
+---                                                                             
+
+# Development Guide                                                                    
+{: .no_toc }                                                                    
+
+## Table of contents                                                            
+{: .no_toc .text-delta }                                                        
+
+1. TOC                                                                          
+{:toc}    
 
-### To return to documentation home page press [here](https://redhatqe.github.io/teflo_terraform_plugin/index.html)
 
 Hey Developers! If you want to contribute to this repository, please follow below steps:
```

Comment:
I guess we can update the developers guide a bit better. but that can be a separate ticket

Reply:
yes, I didn't modify the original terraform docs.

Reply:
will be fixed in a separate ticket